eclipse/dialog_settings.xml

```diff
@@ -3,7 +3,8 @@
 	<section name="SourceActionDialog.methods">
 		<item value="true" key="IgnoreDefault"/>
 		<item value="false" key="SynchronizedModifier"/>
-		<item value="false" key="Comments"/>
```

Comment:
I prefer to leave Comments off by default. Not usually necessary on the `toString()` method.